### ISSUE TITLE
Can storage.net support webhdfs

### ISSUE THREAD

**Issue by iron9light** [aloneguid/storage#194]
If yes, any link?
If no, I do think a lot of guys want it. Any chellange to implement?

**Comment by aloneguid** [comment]
Yes I can't see a reason why it wouldn't work. Feel free to add support for it.
